src/calliope/schemas/config_schema.py

```diff
@@ -190,6 +190,9 @@ class Solve(CalliopeBaseModel):
     solver: str = Field(default="cbc")
     """Solver to use. Any solvers that have Pyomo interfaces can be used. Refer to the Pyomo documentation for the latest list."""
 
+    enable_postprocessing: bool = False
+    """Whether or not to run postprocessing functions."""
+
     zero_threshold: float = Field(default=1e-10)
```

Comment:
Update docstring of zero_threshold to clarify that it is run whether or not postprocessing is enabled and that only setting it to a value of zero will disable it?